.github/workflows/main/installDependencies.py

```diff
@@ -79,10 +84,16 @@
 
 args = parser.parse_args()
 
+archiveURL = string.Template( args.archiveURL ).substitute(
```

Comment:
Why `Template.substitute()` rather than a regular `format()` call? We've got a history of using all the different mechanisms that python provides, but I think `format()` is good enough that we can just standardise on that now.